gitclone/checkout_helper.go

```diff
@@ -26,6 +37,7 @@ func (t fetchOptions) IsFullDepth() bool {
 }
 
 const branchRefPrefix = "refs/heads/"
+const forkRemoteName = "fork"
```

Comment:
i suggest moving this close to the `defaultRemoteName` const (gitclone.go ln 35) as they are related.

Reply:
I also would rename `defaultRemoteName` to `originRemoteName`, since in the case of a fork, we do the naming based on the stored remote name (rather than which one is default or not).